src/main/org/nlogo/app/DirtyMonitor.scala

```diff
@@ -62,7 +66,7 @@ with ModelSavedEvent.Handler
 
   /// how we get clean
   def handle(e: ModelSavedEvent) {
-    dirty(false)
+    dirty(false, true)
```

Comment:
this might be a good place to use parameter names to make the code a bit easier to read: `dirty(false, lockTitleBar = true)`